spec/till_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'till'
+
+describe Till do
+  it{is_expected.to respond_to :place_order}
+
+  before(:each) do 
+    subject.place_order "Cafe Latte"
+    subject.place_order "Cafe Latte"
+  end 
+
+  it 'can collect orders' do
+    expect(subject.customer_order).to eq [{"Cafe Latte"=>4.75}, {"Cafe Latte"=>4.75}]
```

Comment:
Line is too long. [85/80]<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.